backend.native/tests/codegen/branching/empty_nothing_block.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package codegen.branching.empty_nothing_block
+
+import kotlin.test.*
+
+@Test fun runTest() {
+    if (true) else true
```

Comment:
IR produced for this code doesn't seem correct, so there is not much sense in supporting it in code generator.

Reply:
``` if (false) ;``` gives the same problematic code.